bedrock/mozorg/templates/mozorg/home/home-new.html

```diff
@@ -184,7 +184,10 @@ <h2>{{ ftl('home-mozilla-makes-privacy') }}</h2>
     ) %}
     <h2>{{ ftl('home-is-mozilla-a-corporation')}}</h2>
     <p>{{ ftl('home-mozilla-consists-of') }}</p>
-    <a href="{{ url('mozorg.about.manifesto') }}" class="mzp-c-button">{{ ftl('home-learn-about-mofo') }}</a>
+    <a href="https://foundation.mozilla.org/" class="mzp-c-button" rel="external noopener">
+      <span>{{ ftl('home-learn-about-mofo') }}</span>
+      <img src="{{ static('img/home/2023/external-link.png') }}" alt="">
```

Comment:
On hover, the icon remains white. The icon should turn black like the text of the button does.